person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  private
+
+  # I needed to do this is order to keep the method name required
+  # in the project
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    @age >= 18
+  end
+  # rubocop:enable Naming/PredicateName
```

Comment:
- [ ] Good job adding Person, Student and Teacher classes to your project 👏  👏 👏 , however for the sake of linters good practice you should not disable them 👍 ,Kindly delete this comments (#L20. and #L24).

To avoid the rubocop error , Kindly change the name of your method , instead of using `is_of_age` you could use `of_age` 😄 . For more details about that error click [here](https://docs.rubocop.org/rubocop/cops_naming.html)